modules/ROOT/examples/live-demos/accordion/index.html

```diff
@@ -0,0 +1,49 @@
+<textarea id="accordion">
+
+  {{logofordemoshtml}}
+
+  <h1 style="text-align: center;">The Accordions plugin</h1>
+
+  <p>This is a demo of the <strong>Accordion</strong> open source plugin.</p>
+
+  <details>
+    <summary>What does the <strong>Accordion</strong> plugin do?</summary>
+      <p>The Accordion plugin allows for the creation of collapsible sections in a document that can be expanded or collapsed to show or hide additional content.</p>
+  </details>
+
+  <details>
+    <summary>How does it present to an end-user?</summary>
+      <p>By default, when the <code>accordion</code> plugin is loaded, the command is available in the TinyMCE menu at <strong>Insert → Accordion</strong>.</p>
+      <p>Adding <code>accordion</code> to the <code>toolbar</code> in the TinyMCE configuration adds an Accordion icon to the TinyMCE toolbar.</p>
+      <p>And, when a specific collapsible section is selected, a pop-up menu presents offering two controls: one to collaps or expand the selected section; and one to delete the selected section.</p>
+  </details>
+
+  <details>
```

Comment:
Add this attribute to `<details>` tag -  class="mce-accordion"

- Change `<details>` to `<details class="mce-accordion">`

If class is not mentioned then user is unable to select the body content.

FYI @mkzhx

Reply:
So changed.